[2021-05-25 20:08:19,965] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: covid19_cases_Jordan.Get_COVID19_Cases_For_Jordan 2021-05-25T20:07:07.052122+00:00 [queued]>
[2021-05-25 20:08:20,069] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: covid19_cases_Jordan.Get_COVID19_Cases_For_Jordan 2021-05-25T20:07:07.052122+00:00 [queued]>
[2021-05-25 20:08:20,070] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-05-25 20:08:20,071] {taskinstance.py:1043} INFO - Starting attempt 1 of 2
[2021-05-25 20:08:20,072] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-05-25 20:08:20,147] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): Get_COVID19_Cases_For_Jordan> on 2021-05-25T20:07:07.052122+00:00
[2021-05-25 20:08:20,236] {standard_task_runner.py:52} INFO - Started process 863 to run task
[2021-05-25 20:08:20,358] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'covid19_cases_Jordan', 'Get_COVID19_Cases_For_Jordan', '2021-05-25T20:07:07.052122+00:00', '--job-id', '81', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/workflow.py', '--cfg-path', '/tmp/tmp8k0turby', '--error-file', '/tmp/tmpt075hwxn']
[2021-05-25 20:08:20,381] {standard_task_runner.py:77} INFO - Job 81: Subtask Get_COVID19_Cases_For_Jordan
[2021-05-25 20:08:20,976] {logging_mixin.py:104} INFO - Running <TaskInstance: covid19_cases_Jordan.Get_COVID19_Cases_For_Jordan 2021-05-25T20:07:07.052122+00:00 [running]> on host 4e0af5d83701
[2021-05-25 20:08:21,690] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=waed
AIRFLOW_CTX_DAG_ID=covid19_cases_Jordan
AIRFLOW_CTX_TASK_ID=Get_COVID19_Cases_For_Jordan
AIRFLOW_CTX_EXECUTION_DATE=2021-05-25T20:07:07.052122+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-05-25T20:07:07.052122+00:00
[2021-05-25 20:23:57,135] {local_task_job.py:188} WARNING - State of this instance has been externally set to removed. Terminating instance.
[2021-05-25 20:23:57,163] {process_utils.py:100} INFO - Sending Signals.SIGTERM to GPID 863
[2021-05-25 20:23:57,223] {taskinstance.py:1239} ERROR - Received SIGTERM. Terminating subprocesses.
[2021-05-25 20:24:57,192] {process_utils.py:113} WARNING - process psutil.Process(pid=863, name='airflow task ru', status='sleeping', started='20:08:19') did not respond to SIGTERM. Trying SIGKILL
[2021-05-25 20:24:57,579] {process_utils.py:66} INFO - Process psutil.Process(pid=863, name='airflow task ru', status='terminated', exitcode=<Negsignal.SIGKILL: -9>, started='20:08:19') (863) terminated with exit code Negsignal.SIGKILL
